Sources/UIImage+Sweet.swift

```diff
@@ -3,6 +3,26 @@
 
     public extension UIImage {
 
+        /// Creates a QR code from a string.
+        ///
+        /// - Parameter string: Text to be the QR Code content
+        /// - Returns: image QR Code image
+        public static func imageQRCode(for string: String) -> UIImage {
+            let data = string.data(using: .isoLatin1, allowLossyConversion: false)
+
+            let filter = CIFilter(name: "CIQRCodeGenerator")!
+            filter.setDefaults()
+            filter.setValue(data, forKey: "inputMessage")
+            filter.setValue("H", forKey: "inputCorrectionLevel")
+
+            let cImage = filter.outputImage!
+
+            let qrCode = UIImage(ciImage: cImage)
+            let qrCodeResized = qrCode.resize(by: 15.0, quality: .none)
```

Comment:
Can the resize by be injected using 15 as a default value? To avoid having hardcoded things in there.